src/Model/Entity/BussinessObject/Contact.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Friendica\Model\Entity;
+
+use DateTime;
+use Friendica\Model\Entity\Traits\StorableTrait;
+
+/**
+ * @property int      uid
+ * @property DateTime created
```

Comment:
One thing I'd love to do is using "real" datetimes for the PHP calls, so we can use the native PHP-functionallty for it :-)